inst/include/dust/interface.hpp

```diff
@@ -40,6 +44,44 @@ cpp11::list dust_alloc(cpp11::list r_data, int step,
   return cpp11::writable::list({ptr, info});
 }
 
+// think about rng seeding here carefully; should accept either a raw
+// vector or an integer I think.
```

Comment:
What does this mean? Planning to send an rng_state through here rather than a seed, eventually?

Reply:
yes - turns out my ideas on this are still mostly in my head so dumped out some here: #87